backend/inspirehep/accounts/views.py

```diff
@@ -32,14 +32,12 @@ def me():
 
 @blueprint.route("/login_success")
 def login_success():
-    payload = get_current_user_data()
-    return render_template("accounts/postmessage.html", payload=payload)
+    return redirect(session["next_url"])
 
 
 @blueprint.route("/signup", methods=["GET"])
 def sign_up_required():
-    payload = {"user_needs_sign_up": True}
-    return render_template("accounts/postmessage.html", payload=payload)
+    return redirect(session["signup_url"])
```

Comment:
we don't need this, since `signup_url` is static, we can just set them as `signup_redirect_url` in config, rather than passing first from the client and then redirecting here.

Reply:
That URL points to the frontend's SignUpForm. We shouldn't set the URL here because we would be coupling with the frontend

Reply:
I think this wouldn't be the kind of coupling that we care, at the moment at least.

Reply:
Actually that was one of my points when discussing with Miguel, starting adding UI endpoints to backend change more rule in multiple places (i.e. Traefik). It shouldn't be often but still I don't see any value hardcoding it.